src/whylogs/app/logger.py

```diff
@@ -355,6 +360,30 @@ def log_image(self,
 
         track_image(self._profiles[-1]["full_profile"])
 
+    def log_local_dataset(self, root_dir, folder_feature_name="folder_feature"):
+        from PIL.Image import Image as ImageType
+        dst = LocalDataset(root_dir)
+        for idx in tqdm(range(len(dst))):
```

Comment:
I don't think we should use progress bar here. It will interfere with users logging set up. We should rely on standard loggers only if we need to log out debug information

Reply:
I dont think it interferences with the logging, it goes to stdout only.

Reply:
Also this is only used when you log a dataset, which can take a while depending on the size.

Reply:
If you are in Docker, stdout is your logging FYI :)

Maybe put this behind a parameter such as `show_progress_bar`? And it should be an optional dependency imported locally here

Reply:
i will wrap it but I dont like to hide thin requirements. Otherwise it catches folks by surprise in the middle of run time

Reply:
@andyndang done